tests/testsuite/fixit.rs

```diff
@@ -154,6 +156,9 @@ fn build_unit_order() {
     p.cargo_("fixit --allow-no-vcs")
         .with_status(0)
         .with_stderr_data(str![[r#"
+[FIXED] foo v0.1.0
+[FIXED] foo v0.1.0
+[FIXED] foo v0.1.0
```

Comment:
I thought we had discussed what to do about packages vs build targets but not finding it anywhere.  Printing the same message over and over seems like it'd be confusing.  Cargo at least will report this when all build-targets are done.